src/openads-appnexus/AppNexusConnector.js

```diff
@@ -108,15 +108,22 @@ export default class AppNexusConnector {
         )
         if (normalizedInputs.adUnits.length > 0 && this._prebidClient) {
           this._prebidClient.addAdUnits({adUnits: normalizedInputs.adUnits})
-          if (this._prebidConfig)
-            this._prebidClient.setConfig(this._prebidConfig)
+
+          if (this._prebidConfig && this._prebidConfig.core)
+            this._prebidClient.setConfig(this._prebidConfig.core)
```

Comment:
please, use brackets in the if setences to avoid confusion and update errors (if I add another line that affects to the conditional statement, I will need the brackets to take effect or an error would occur)